src/sonic-config-engine/sonic-cfggen

```diff
@@ -211,6 +211,7 @@ def main():
     group.add_argument("--preset", help="generate sample configuration from a preset template", choices=get_available_config())
     group = parser.add_mutually_exclusive_group()
     group.add_argument("-K", "--key", help="Lookup for a specific key")
+    parser.add_argument("-n", "--namespace", help="Namespace string to use asic0/asic1.../asicn", default=None)
```

Comment:
Could you also explain the ignore behavior, and also the `-namespace=''` behavior in usage?

Reply:
This is a simple suggestion on `help` text.

---
In reply to: [416108613](https://github.com/Azure/sonic-buildimage/pull/4477#discussion_r416108613) [](ancestors = 416108613)